### PR TITLE
Stress

### PR DESCRIPTION
**Pull request by joshskkim**
@RetailHub/gandalf Reconfigured stress testing routes

### CODE REVIEW

**Comment by sajjanrajvaidya** [comment]
dope

File: server/test.yml [hunk 1]
```diff
@@ -19,19 +19,25 @@ config:
...
-        json:
-          itemId: "{{itemId}}"
-          altImages: "{{altImages}}"
-    - log: "Posted {{itemId}}, {{altImages}}"
+  - name: "Checking out pages"
+    weight: 90
+    flow:
+      - function: "generateRandomPages"
+      - get:
+          url: "/api/items/{{id}}"
```
Comment (sajjanrajvaidya) [inline server/test.yml:32]: What is happening here? Is it sending variable IDs on its own?

Comment (joshskkim) [reply server/test.yml:32]: takes the randomly generated ID from the imported function

Comment (sajjanrajvaidya) [reply server/test.yml:32]: That is so sick!

Comment (joshskkim) [reply server/test.yml:32]: It is pretty dope!!!

**Pull request merged**